core-bundle/src/Controller/SitemapController.php

```diff
@@ -140,7 +140,7 @@ private function getPageAndArticleUrls(int $parentPageId): array
                 continue;
             }
 
-            $isPublished = $pageModel->published && (!$pageModel->start || $pageModel->start <= time()) && (!$pageModel->stop || $pageModel->stop > time());
+            $isPublished = $pageModel->published && (!$pageModel->start || $pageModel->start <= time()) && (!$pageModel->stop || $pageModel->stop > time()) && 'noindex,nofollow' !== $pageModel->robots;
```

Comment:
It is wrong to consider the meta robots setting to determine the `isPublished` status, isn't it?

Reply:
Should we rename the Variable to something like `$isPublishedForSitemap` to make it more clear?

Reply:
True. I can move it down to line 145 instead. Then it would be more or less in the "same"-ish spot as before: 

https://github.com/contao/contao/blob/bdc6e30cf50421bf3ec98d67b0b21ddd5ffa950d/core-bundle/src/Resources/contao/classes/Backend.php#L695-L698